README.md

```diff
@@ -435,8 +435,8 @@ if the HTTP calls becomes successful, they will begin to send immediately, and w
 
 * The client is optimized for high throughput. Using a single instance is no problem, but multiple instances can be created with the same key if desired.
 * Set up an `error` listener so that your app is aware of problems. Things like HTTP errors are emitted this way.
-* When shutting down your application, ensure all log entries are cleared. Services like AWS Lambda can buffer log entries,
-  so it might be worthwhile to pause for a short time before exiting like in the following example:
```

Comment:
Removed the references here to Lambda - any buffering of `console.log` isn't really relevant here, and the only thing needed is waiting for the final `flush` to clear.